src/components/POLine/POLineInvoices/POLineInvoices.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import {
+  find,
+  get,
+} from 'lodash';
+
+import { MultiColumnList } from '@folio/stripes/components';
+
+import FolioFormattedDate from '../../FolioFormattedDate';
+
+const POLineInvoices = ({ locale, lineInvoices, vendors }) => {
+  const resultFormatter = {
+    'invoice': invoice => (
+      <Link to={`/invoices/view/${invoice.id}`}>
+        {get(invoice, 'folioInvoiceNo', '')}
+      </Link>
+    ),
+    'invoiceDate': invoice => <FolioFormattedDate value={get(invoice, 'invoiceDate')} />,
+    'vendorName': invoice => get(find(vendors, ['id', get(invoice, 'vendorId', '')]), 'name', ''),
+    'status': invoice => get(invoice, 'status', ''),
+    'expendedAmount': invoice => (
+      Intl.NumberFormat(locale, { style: 'currency', currency: get(invoice, 'currency', '') }).format(get(invoice, 'total', ''))
```

Comment:
Please leverage this into a component. You can use `withStripes()` hoc to provide stripes and locale there. I think it's a good idea to put that component to stripes-acq-components

Reply:
Done